solace-deployment.yml

```diff
@@ -1835,6 +1845,8 @@ variables:
   type: password
 - name: mgmt_password
   type: password
+- name: monitor_password
+  type: password
```

Comment:
We don't want bosh to generate a random password for us, we want to use the password from the config file.